main.go

```diff
@@ -125,7 +126,7 @@ func mainE() error {
 			return fmt.Errorf("checkout (%s): %v", checkoutArg, err)
 		}
 		// Update branch: 'git fetch' followed by a 'git merge' is the same as 'git pull'.
-		if checkoutArg == cfg.Branch {
+		if checkoutArg == cfg.Branch && cfg.AutoMerge{
```

Comment:
Not sure if we should ever do merge origin/master after a fetch and checkout?

Seems to work fine